### ISSUE TITLE
Needs android namespace for Gradle 8 support

### ISSUE THREAD

**Issue by cpboyd** [pikaju/flutter-maps-launcher#31]
## Bug report

**Describe the bug**
```
A problem occurred configuring project ':firebase_auth'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Please specify a namespace in the module's build.gradle file like so:
android {
    namespace 'com.example.namespace'
}
```

### Steps to reproduce

1. Run AGP in latest Android Studio

### Expected behavior

Should be compatible with the version of Gradle suggested by Android Studio.

**Comment by GandhiManan** [comment]
Is this issue resolved? I have used all 8.x versions of gradle

**Comment by mananrg** [comment]
I have used all 8.x versions of gradle any solutions to this issue?

**Comment by AuroreT** [comment]
Apparently, namespace in build.gradle is missing:
`A problem occurred configuring project ':maps_launcher'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.`

**Comment by cpboyd** [comment]
It seems like this should've been fixed with #32
I'd manually stubbed the local copies of plugins with the 
```
android {
    namespace 'com.example.namespace'
}
```

If you're having issues, you could try that or open a new issue.

**Comment by gourgouris** [comment]
I've opened a new issue here: (https://github.com/pikaju/flutter-maps-launcher/issues/40)
